hi.js

```diff
@@ -858,21 +858,21 @@ stats.assists असिस्ट
 stats.nukes न्यूक
 stats.slimers Slimers
 stats.juggernauts Juggernauts
-stats.juggernauts.kills Juggernaut Kills
-stats.juggernauts.killed Juggernauts Killed
+stats.juggernauts.kills Juggernaut किल्स 
+stats.juggernauts.killed Juggernauts किल्ड 
 stats.warmachines War Machines
 stats.melee चाकू किल
 stats.fistkills हाँथों द्वारा किये गए किल
 stats.thrownkills फेंके गए चाक़ू द्वारा किल
 stats.headshots हेड शाट
 stats.wallbangs वॉल बैंग
-stats.crouches Crouches
+stats.crouches क्रोचेस 
 stats.sprays स्प्रे काउन्ट
 stats.kpd मददगार रिपोर्टस
-stats.shots Shots
-stats.hits Hits
-stats.misses Misses
-stats.legshots Legshots
+stats.shots शॉट्स 
+stats.hits हिट्स 
+stats.misses मिसेस 
+stats.legshots लेग्शॉट्स 
```

Comment:
लेग्शॉट्स will feel complicated. Keep it as it is . P.S. - लेगशाट्स is the easier translation here